lib/plsql/connection.rb

```diff
@@ -197,9 +197,8 @@ def session_id
       @session_id ||= select_first("SELECT TO_NUMBER(USERENV('SESSIONID')) FROM dual")[0]
     end
 
-    # Set time zone (default taken from TZ environment variable)
+    # Set time zone
     def set_time_zone(time_zone=nil)
-      time_zone ||= ENV['TZ']
```

Comment:
True, will do cleanup of those calls.
The nil as default doesn't seem to make sense anymore.
however LOCAL is a keyword not a TZ name, and again setting to LOCAL makes the dumb thing, setting timezone as a fixed offset.